src/views/forms/AccountManagementFormsNew/HighestFeesText.tsx

```diff
@@ -0,0 +1,6 @@
+import { Tag, TagSign } from '@/components/Tag';
+
+// TODO: add localization
```

Comment:
nit: you can suffix your TODOs with your project name so its easier to search for all TODOs before you launch
```suggestion
// TODO(deposit-refresh): add localization
```